src/bridging/providers/across/const/tokens.ts

```diff
@@ -57,6 +57,10 @@ const ACROSS_CHAIN_CONFIGS: AcrossChainConfig[] = [
       usdt: '0x94b008aA00579c1307B0EF2c499aD98a8ce58e58',
     },
   },
+  {
+    chainId: SupportedChainId.AVALANCHE,
+    tokens: {}, // ACROSS does not support AVALANCHE (yet)
+  },
```

Comment:
In that case, might as well not add it to this list?
For example, Gnosis chain is not here either.